services/api/installations.py

```
@@ -434,18 +435,22 @@ def _lookup(token: str) -> tuple[bool, str, str]:
+            updated_by_user_id=user.github_user_id,
+            guard_enabled=body.guard_enabled,
+            guard_blocking=body.guard_blocking,
+            warder_enabled=body.warder_enabled,
+            pulse_enabled=body.pulse_enabled,
+            smasher_enabled=body.smasher_enabled,
+            walkthrough_enabled=body.walkthrough_enabled,
+            dep_watch_enabled=body.dep_watch_enabled,
+            elder_voice=body.elder_voice,
+        )
+    except ValueError as e:
+        raise HTTPException(status_code=status.HTTP_403_FORBIDDEN, detail=str(e))
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)